languages.json

```
@@ -937,6 +937,14 @@
       "quotes": [["\\\"", "\\\""], ["'", "'"]],
       "extensions": ["nqp"]
     },
+    "Nushell": {
+      "line_comment": ["#"],
+      "quotes": [
+        ["\\\"", "\\\""],
+        ["'", "'"],
```

You have a trailing comma here. I'm also going to make an issue about switching this file to json5 as it's not contributor friendly to have a PR fail because of a comma

```suggestion
        ["'", "'"]
```